tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -429,7 +435,7 @@ public function testCreateExplicitMethodEnctype(): void
             'enctype' => 'multipart/form-data',
         ]);
         $expected = ['form' => [
-            'method' => 'put',
+            'method' => 'post',
```

Comment:
This was an incorrect assertion since `put` is not a value value for the `form` tag's `method` attribute.